pkg/tsdb/influxdb/response_parser.go

```diff
@@ -122,7 +122,7 @@ func transformRows(rows []Row, query Query) data.Frames {
 				}
 			}
 
-			field := data.NewField("value", nil, values)
+			field := data.NewField("Value", nil, values)
```

Comment:
We already had existing dashboards breaking because of the different capitalization of a field name (`time`) 
 between versions. 

Are we sure that this will not break anything ?